src/components/sbb-loading-indicator/sbb-loading-indicator.scss

```diff
@@ -0,0 +1,235 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host([variant='circle']) {
+  --sbb-loading-indicator-height: var(--sbb-size-icon-ui-small);
```

Comment:
According to the specs, the dimension of the circle should "adjusts the size to the font size". From what I see here and from a quick test, it doesn't adapt to the inline text but it's always `--sbb-size-icon-ui-small`.

Reply:
Done in [7c71e96](https://github.com/lyne-design-system/lyne-components/pull/1968/commits/7c71e96ad0e0d85ecab9bfba1c200e0164e98130)